### ISSUE TITLE
Uncaught SyntaxError: Unexpected token '<'

### ISSUE THREAD

**Issue by sharpapp-mareklanduch** [SheetJS/ssf#41]
Hi

I ran "yarn add ssf". I haven't done any imports of this library in my existing project just ran "yarn add ssf". After that I ran"yarn install" to install all dependencies and then "yarn start".
No compilation error just "Uncaught SyntaxError: Unexpected token '<'" in chrome console
![image](https://user-images.githubusercontent.com/5566853/76327173-f2e1ae80-62e9-11ea-958d-aac372360c4b.png)

Tried on react-scripts versions 3.4.0 - not working,
3.3.1 - not working,
3.2.0 **WORKING** :)

So if react-scripts in version 3.2.0 works just fine is it something that you guys could fix or is it something wrong with react-scripts :D 

Because of this error library xlsx is not working since ssf is one of it's dependencies

**Comment by sharpapp-mareklanduch** [comment]
It turned out that frac is causing this error I rised issue in other repo https://github.com/SheetJS/frac/issues/4

**Comment by sharpapp-mareklanduch** [comment]
https://github.com/facebook/create-react-app/issues/8490 - this thread helped me to solve my issue

I had unexpected token because I used PUBLIC_PATH variable in .env file
PUBLIC_PATH:/my-app/
I didn't have this variable in .env.development file.
So in react-scripts 3.2.0 it was by default "/" and after I updated react scripts it was failing with unexpected token error. I simply added PUBLIC_PATH:/ to my .env.development file.
